services/workflows-service/docker-compose.redis.yml

```diff
@@ -0,0 +1,26 @@
+version: '3.8'
+services:
+  redis:
+    image: redis:alpine
```

Comment:
Is it for local only? Because the Alpine image is slower than the Debian image

Reply:
yes it's local only